rocky/katalogus/views/plugin_enable_disable.py

```diff
@@ -26,19 +26,26 @@ def post(self, request, *args, **kwargs):
                 _("{} '{}' enabled.").format(self.plugin.type.title(), self.plugin.name),
             )
         else:
-            member_clearance_level_text = (
-                f"Your clearance level is L{self.organization_member.acknowledged_clearance_level}. Contact your "
-                f"administrator to get a higher clearance level."
-            )
-
             if (
-                self.organization_member.trusted_clearance_level < 0
-                or self.organization_member.acknowledged_clearance_level < 0
+                self.organization_member.trusted_clearance_level
+                != self.organization_member.acknowledged_clearance_level
             ):
+                member_clearance_level_text = _(
+                    "You have not acknowledged your clearance level. "
+                    "Go to your profile page to acknowledge your clearance level."
+                )
+            elif self.organization_member.max_clearance_level < 0:
                 member_clearance_level_text = _(
                     "Your clearance level is not set. Go to your profile page to see your clearance "
                     "or contact the administrator to set a clearance level."
                 )
+            else:
+                clearance_level = self.organization_member.max_clearance_level
+
+                member_clearance_level_text = _(
+                    f"Your clearance level is L{clearance_level}. Contact your "
+                    f"administrator to get a higher clearance level."
```

Comment:
I think for correct translations and avoiding duplicates you'd be better of using a `%d` format here for `clearance_level` instead of an f-string or any pre-formatted string